index.dd

```diff
@@ -392,15 +392,16 @@ $(DIVC frontpage-orgs center,
 )
     <div id="frontpage-orgs-more">
         <i class="fa fa-angle-double-right" aria-hidden="true"></i>
+        $(LINK2 $(ROOT_DIR)areas-of-d-usage.html, Current D use)
+        $(BR)
+        <i class="fa fa-angle-double-right" aria-hidden="true"></i>
```

Comment:
> I guess you could argue about it either way?

Maybe a native speaker has an opinion on this?

> Perhaps we should

Yeah consistency is more important than ultimate "correctness" ;-)